functions-methods.js

```diff
@@ -20,6 +26,24 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail (email){
+    let emailAdres=email.toString();
+    let answer;
+    if(getEmailDomain(emailAdres)==="novi-education.nl"){
+        answer = "Student";
+    } else if (getEmailDomain(emailAdres)==="novi.nl") {
+        answer = "Medewerker";
+    } else {
+        answer = "Extern"
+    }
+    return answer
```

Comment:
Je zou ook meteen de waarde terug kunnen geven
```suggestion
    if(getEmailDomain(emailAdres) === "novi-education.nl"){
        return "Student";
    } else if (getEmailDomain(emailAdres) === "novi.nl") {
        return "Medewerker";
    } else {
        return "Extern";
    }
```